source/jormungandr/jormungandr/instance.py

```diff
@@ -134,6 +134,7 @@ def __init__(
         self.georef = georef.Kraken(self)
         self.planner = planner.Kraken(self)
         self._streetnetwork_backend_manager = streetnetwork_backend_manager
+        self.asynchronous_ridesharing = False
```

Comment:
You are a brave man ! 
I would have been just fine with `async_`, but you are not the kind of guy that takes shortcuts :laughing: